credential/manifest/validation.go

```diff
@@ -16,41 +16,43 @@ import (
 // IsValidCredentialApplicationForManifest validates the rules on how a credential manifest [cm] and credential
 // application [ca] relate to each other https://identity.foundation/credential-manifest/#credential-application
 // applicationAndCredsJSON is the credential application and credentials as a JSON object
-func IsValidCredentialApplicationForManifest(cm CredentialManifest, applicationAndCredsJSON map[string]interface{}) (unfulfilledInputDescriptors map[string]string, err error) {
+func IsValidCredentialApplicationForManifest(cm CredentialManifest, applicationAndCredsJSON map[string]interface{}) (map[string]string, error) {
```

Comment:
I used named return here because I thought that a map[string]string on its own might not be clear. I agree this is a cleaner impl though

Reply:
This was mostly to make the `bare-return` happy.